nibabel/_version.py

```diff
@@ -284,7 +284,7 @@ def git_pieces_from_vcs(tag_prefix, root, verbose, run_command=run_command):
         # TAG-NUM-gHEX
         mo = re.search(r'^(.+)-(\d+)-g([0-9a-f]+)$', git_describe)
         if not mo:
-            # unparseable. Maybe git-describe is misbehaving?
+            # unparsable. Maybe git-describe is misbehaving?
```

Comment:
We should fix issues in Versioneer in https://github.com/python-versioneer/python-versioneer, rather than making edits here.

Reply:
Done.

Reply:
Note that this has already been fixed in Versioneer, just 14 daus ago:
https://github.com/python-versioneer/python-versioneer/pull/260